resources/templates/asg-cdk/deploy.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+echo "Provisioning ASG resources"
+echo "FAIL" > deploy-status.txt
```

Comment:
What's with the "FAIL" log and then "OK" log after? Just making sure the "FAIL" makes sense here.

Reply:
Talked in slack… we are good!